comps/guardrails/pii_detection/pii/pii_utils.py

```diff
@@ -71,7 +73,26 @@ def detect_pii(self, text):
 
 class PIIDetectorWithML(PIIDetector):
     def __init__(self):
+        import joblib
+        from huggingface_hub import hf_hub_download
+        from sentence_transformers import SentenceTransformer
+
         super().__init__()
+        print("Loading embedding model...")
+        embed_model_id = "nomic-ai/nomic-embed-text-v1"
+        self.model = SentenceTransformer(model_name_or_path=embed_model_id, trust_remote_code=True)
+
+        print("Loading classifier model...")
+        REPO_ID = "Intel/business_safety_logistic_regression_classifier"
+        FILENAME = "lr_clf.joblib"
```

Comment:
Do we need intel authorized token to access this model?

Reply:
The model will be made public so the users don't need intel credential

Reply:
The model has been made public so users do not need access authorization.